core/store/tx_manager.go

```diff
@@ -130,18 +134,18 @@ func (txm *EthTxManager) OnNewHead(*models.Head) {}
 
 // CreateTx signs and sends a transaction to the Ethereum blockchain.
 func (txm *EthTxManager) CreateTx(to common.Address, data []byte) (*models.Tx, error) {
-	return txm.CreateTxWithGas(to, data, txm.config.EthGasPriceDefault(), DefaultGasLimit)
+	return txm.CreateTxWithGas(null.String{}, to, data, txm.config.EthGasPriceDefault(), DefaultGasLimit)
```

Comment:
Conceptually I'm confused as to when we would ever want to `CreateTx` without a surrogate ID, and why the presence of gas makes that scenario different. Is it just that this method isn't used in areas of concern? Can we remove it?

Reply:
We have an endpoint that allows you to create transactions in the Withdrawals Controller that needs this.